conf/spotinst-integration/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  # required_version = ">= 1.0.0"
```

Comment:
why this is commented out?

Reply:
Commented out because I do not know what the minimum required version of Terraform is, but it is surely less than 1.0